yfinance/data.py

```diff
@@ -358,7 +358,7 @@ def get(self, url, user_agent_headers=None, params=None, proxy=None, timeout=30)
 
         request_args = {
             'url': url,
-            'params': params | crumbs,
```

Comment:
@ValueRaider 
`|` is not supported and throws `unsupported operand type(s) for |: 'dict' and 'dict` when I try to make a request